src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -294,6 +377,7 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                     BasisStateLabelingConvention.BigEndian => " (big endian)",
                     _ => ""
                 };
+                //TODO: rewrite all this code here for less code written
```

Comment:
Looks like a remaining TODO here as well?

Reply:
Fixed this! thanks!